### DISCUSSION TITLE
[Bug] After upgrading to Firefox 133.0, the horizontal bar of tabs, which I've successfully hid with chrome.css, has reappeared

### DISCUSSION THREAD

**Discussion by hub2git** [piroor/treestyletab#3676] [Q&A] [answered]
I've upgraded from one firefox version to another, and haven't had this problem. It's just today, when I upgraded to version 133.0.

This is my chrome.css
```
#titlebar {
display: none !important;
}
 
#sidebar-box[sidebarcommand="treestyletab_piro_sakura_ne_jp-sidebar-action"] #sidebar-header {
  display: none;
}    
 
#sidebar {
  min-width: 100px !important;
}
 
```
## Expected result
The horizontal strip of tabs should stay hidden, as it has been for many previous versions.

## Environment

 * Platform (OS):  Linux Mint Cinnamon 64-bit
 * Version of Firefox: 133.0 (64-bit)
 * Version (or revision) of Tree Style Tab: 4.0.25 (Nov 19, 2024)x

![screenshot_TreeStyleTab_horizontal_tabs](https://github.com/user-attachments/assets/3bcd14fd-f727-4218-a172-29742a7938e7)

**Comment by piroor** [comment]
I've moved this from issues to discussions, because the issue tracker concentrates to track problems of TST itself and userChrome.css customization  issue is out of purpose. Thanks.

**Comment by irvinm** [comment, marked as answer]
```
#main-window #TabsToolbar {
   display: none;
}  
```

OR you can use https://addons.mozilla.org/en-US/firefox/addon/toggle-native-tab-bar/ to do it dynamically.

**Reply by hub2git** [reply]
Thanks @irvinm . I used your code and it does what I wish.

May I ask you... where did you find that snippet of code? Was that something that became necessary only with version 133 of Firefox?

**Reply by irvinm** [reply]
There was a fair amount of discussion on reddit on these changes while FF133 was in dev & beta.  FF133 changed the elements, so the CSS had to be updated.  There are different ways to do this, but this was my solution I modified from the original solutions.
